app/controllers/claims/support/schools/providers_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Claims::Support::Schools::ProvidersController < Claims::Support::ApplicationController
```

Comment:
Does the Claims service need a Providers controller? 🤔

I was of the understanding that Providers weren't a first-class entity in Track & Pay?

Reply:
I'm referencing the current prototype here: https://itt-mentoring-beta-94341ca35a2c.herokuapp.com/support/organisations/049fc44e-6876-4884-8a77-d363173ce8d2#

If needed, we can drop it when it's removed from the prototype.

Reply:
I think this is the kind of thing that concerns me -- will we remember to drop it? Or will we just create the necessary thing at the time it is needed and this will just live forever, a red herring for someone trying to debug or understand our codebase in the future?  But again, don't feel strongly about it.